builder/bundles.go

```diff
@@ -267,8 +267,11 @@ func parseNoopInstall(installOut string) ([]packageMetadata, error) {
 		// If there is no such section, e.g. dnf fails because there's
 		// no matching package (so no "Installing:"), return nil. Real
```

Comment:
Can you please update the legacy comment to reflect the current behavior.

Reply:
updated